### DISCUSSION TITLE
More documentation on citation-abbreviation functionality

### DISCUSSION THREAD

**Discussion by blenback** [quarto-dev/quarto-web#560] [General]
Hi, 
I am trying to prepare an academic paper with Quarto, I have a in-text citation which I want to replace with an abbreviated form:
i.e. (Swiss Federal Oﬀice for Statistics (SFSO) ,2021) to (SFSO 2021).  But I want my bibliography to still contain the unabbreviated form. 

I have found the documentation for the YAML option: citation-abbreviations, which says the following: 

citation-abbreviations: JSON file containing abbreviations of journals that should be used in formatted bibliographies when form="short" is specified. 

I have created a JSON file in the style provided as an example and included it in the YAML but I see no change. What I can't find is the option form="short" where does this need to be specified in the YAML or in the .bib file? Some more detail on this in the description would be great, Thanks!

**Comment by dragonstyle** [comment]
I think believe that the `form="short"` is a property  set on nodes of the CSL schema (e.g. the schema itself for rendering a term would declare that it prefers the short form which will be loaded from the JSON abbreviations list). If you are using a CSL file to produce the formatted reference, I think that you should start by trying to tweak that to provide the formatting that you're after.
